src/frontend/index/tabs.nim

```diff
@@ -1,8 +1,11 @@
 import
```

Comment:
maybe this can be unified with files.nim: the function naming is just historical, but they usually work with the same things here: sources/files shown in tabs or other places